### ISSUE TITLE
Make explicit SSSOM-py uses zero-versioning

### ISSUE THREAD

**Issue by cthoyt** [mapping-commons/sssom-py#476]
version 1.0.0 is scary and scientists can't commit to it so we just never get to 1.0 and use minor versions as major versions. This is called zero versioning, and is useful to explicitly communicate: https://0ver.org/

**Comment by matentzn** [comment]
Great point. What is the best practice to "announce" this? Just a fat line on the README?

**Comment by cthoyt** [comment]
yes. Zero version is also half-satire, so not sure what you think about that

**Comment by matentzn** [comment]
I find it realistic more than satire :P

**Comment by cthoyt** [comment]
That's what makes it good satire ;)
